eli5/sklearn/utils.py

```diff
@@ -35,7 +35,7 @@ def get_feature_names(clf, vec=None, bias_name='<BIAS>', feature_names=None):
     """
     if feature_names is None:
         if vec and hasattr(vec, 'get_feature_names'):
-            feature_names = vec.get_feature_names()
+            feature_names = list(vec.get_feature_names())
```

Comment:
Forgot to add - this change is required because for some some vectorizers (DictVectorizer) get_features_names returns a list that holds the feature names for the vectorizer, and we append to it bias several lines below and modify it.